CodeHS/2/17/Happy-Sad-Face-Travis.py

```diff
@@ -0,0 +1,53 @@
+speed(0)
```

Comment:
You're missing a directory, i think you meant to use 
`CodeHS/2/17/5`
you used
`CodeHS/2/17`